api/src/main/java/org/apache/cloudstack/api/command/user/offering/ListServiceOfferingsCmd.java

```diff
@@ -157,14 +154,7 @@ public String getStorageType() {
     }
 
     public State getState() {
-        if (StringUtils.isBlank(serviceOfferingState)) {
-            return Active;
-        }
-        State state = EnumUtils.getEnumIgnoreCase(State.class, serviceOfferingState);
-        if (!serviceOfferingState.equalsIgnoreCase("all") && state == null) {
```

Comment:
Same here

Reply:
I just made this uniform to how it works with listing network and vpc offerings ; wherein when the state isn't passed, we list all offerings - so we do not explicitly need "all"